### PR TITLE
PubMatic to support DigiTrust Id passing

### PR DESCRIPTION
**Pull request by pm-harshad-mane**
## Type of change
- [x] Feature

## Description of change
PubMatic adapter to pass DigiTrust Id

### CODE REVIEW

**Comment by pm-harshad-mane** [comment]
Hello @jsnellbaker and @mkendall07 
Could you please review?

File: modules/pubmaticBidAdapter.js [hunk 1]
```diff
@@ -275,6 +277,46 @@ function _createImpressionObject(bid, conf) {
...
+    });
+  }
+}
+
+function _handleEids(payload) {
+  let eids = [];
+  _handleDigitrustId(eids);
+  if (eids.length > 0) {
+    // todo: are we sure we have to send this data at user.eids or user.ext.each_eid_solution_object?
+    payload.user.eids = eids;
```
Comment (jsnellbaker) [inline modules/pubmaticBidAdapter.js]: To clarify - is this still an open question?  Or has it been confirmed?

If the latter, can you remove the comment?

File: test/spec/modules/pubmaticBidAdapter_spec.js [hunk 2]
```diff
@@ -448,6 +449,197 @@ describe('PubMatic adapter', function () {
   		  expect(data.imp[0].ext.pmZoneId).to.equal(bidRequests[0].params.pmzoneid.split(',').slice(0, 50).map(id => id.trim()).join()); // pmzoneid
   		});
 
+      it('Request should have digitrust params', () => {
```
Comment (jsnellbaker) [inline test/spec/modules/pubmaticBidAdapter_spec.js]: Can you update the arrow functions in the various test functions to use the `, function() {` syntax?  This is more in-line with Mocha's style/expectations.

File: modules/pubmaticBidAdapter.js (hunk 1 above)
Comment (pm-harshad-mane) [reply modules/pubmaticBidAdapter.js]: Hello @jsnellbaker , 
I have removed the comment now.

File: test/spec/modules/pubmaticBidAdapter_spec.js (hunk 2 above)
Comment (pm-harshad-mane) [reply test/spec/modules/pubmaticBidAdapter_spec.js]: I have made these changes now.

**Comment by pm-harshad-mane** [comment]
Hello @jsnellbaker ,
I have made the requested changes.

**Pull request merged**